src/server.tsx

```diff
@@ -381,7 +300,7 @@ app.get(
 );
 
 // TODO remove once migration complete
-app.post('/epic/compare-variant-decision', async (req: express.Request, res: express.Response) => {
+/* app.post('/epic/compare-variant-decision', async (req: express.Request, res: express.Response) => {
```

Comment:
🔥 ?

Reply:
Thanks. I was planning to fix this endpoint but forgot - will update.